base/shared/src/main/scala/scalaz/ct/package.scala

```diff
@@ -7,4 +7,6 @@ package object ct {
 
   val Kleisli: KleisliModule = KleisliImpl
   type Kleisli[F[_], A, B] = Kleisli.Kleisli[F, A, B]
+  val ReaderT: Kleisli.type = Kleisli
+  type ReaderT[F[_], A, B] = Kleisli[F, A, B]
```

Comment:
@TomasMikula Isn't this making the work I did for 7.3.x irrelevant?

Reply:
Seems the scalaz8 authors didn't get the memo :-P

Reply:
Why does this relate to 7.3.x, @JordanMartinez?

Reply:
@edmundnoble See #1305

Reply:
There is a
```haskell
instance Functor f => Profunctor (Kleisli f)
```
which won't resolve with a different order of type parameters. I guess it makes sense for transformers.

Reply:
I think we should do this for all monad transformers in 8, @JordanMartinez and @alexknvl. I won't do it in this PR, though.

Reply:
I can take care of that in a later PR. Though I'll learn from the previous time I did that and first clarify all of the work it entails before doing it, so that something as simple as this doesn't take 2 weeks like last time.